plugins/woocommerce-blocks/assets/js/base/components/drawer/style.scss

```diff
@@ -112,7 +117,7 @@ $drawer-animation-duration: 0.3s;
 }
 
 // Important rules are needed to reset button styles.
-.wc-block-components-drawer__close {
+.wc-block-components-button.wc-block-components-drawer__close {
```

Comment:
To ensure specificity I've appended the button class. In future I think we could refactor the base button styles, but it's a lot more risk and complexity to do that in the scope of this PR and it would have wide-ranging style implications.